templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig

```diff
@@ -1,110 +1,88 @@
- *   - social_media_title: Title for social media links column.
- *   - social_media: Social media links for corporate footer.
- *   - institution_links_title: Title for institution links column.
- *   - institution_links: EU institutions links for corporate footer.
- *   - legal_links_title: Title for legal links column.
- *   - legal_links: Legal links for corporate footer.
- *
- * @ingroup themeable
- */
-#}
-
-{% set _section_2 = [] %}
-{% set _section_3 = [] %}
-
-{# First we parse the other_links, we make a simple grid. #}
-{% for section in site_specific_footer.other_links %}
-  {% if loop.index is odd %}
-    {% set _section_2 = _section_2|merge([{
-      'title': section.label,
-      'links': bcl_footer_links(section.links),
-    }]) %}
-  {% else %}
-    {% set _section_3 = _section_3|merge([{
-      'title': section.label,
-      'links': bcl_footer_links(section.links),
-    }]) %}
-  {% endif %}
-{% endfor %}
-
-{# When follow section is present. #}
-{% if site_specific_footer.social_links is not empty %}
-  {# If section numbers are odd, then we flip the last row so to have the follow as first. #}
-  {% if _section_2|length > _section_3|length %}
-    {% set _last_in_section_2 = _section_2|last %}
-
-    {# Compensate for bug in twig where _section_2[1:] returns empty if _section_2|length is 1. #}
-    {% if _last_in_section_2 is empty %}
-      {% set _last_in_section_2 = _section_2[0] %}
-      {% set _section_2 = [] %}
-    {% endif %}
-
-    {% set _section_2 = _section_2[:_section_2|length - 1] %}
-    {% set _section_3 = _section_3|merge([_last_in_section_2]) %}
-
-  {% endif %}
-
-  {# Follow section must always be the last on the left. #}
-  {% set _section_2 = _section_2|merge([{
-    'title': 'Follow us on social media'|t,
-    'links': bcl_footer_links(site_specific_footer.social_links),
-  }]) %}
-{% endif %}
+{% extends "@oe_whitelabel/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig" %}
+{% block content %}
+  {% set _top_sections =  {
+    'extra_classes': 'pt-4 pt-lg-5',
+    'cols': [
+      {
+        'classes': 'col-12 col-lg-4',
+        'items': [
+        {
+          'type': 'content',
+          'content': footer_top_left_content,
+        },
+      ],
+      },
+      {
+        'classes': 'col-12 col-lg-4 pb-4 pb-lg-0',
+        'items': _section_2,
+      },
+      {
+        'classes': 'col-12 col-lg-4',
+        'items': _section_3,
+      }
+    ],
+  } %}
 
-{% set _top_sections = [
-  {
-    'section_id': 'footer_top_left',
-    'title': {
-      'link': {
-        'label': site_specific_footer.site_identity,
-        'path': url('<front>')|render
+  {% set _bottom_sections =  {
+    'extra_classes': 'pb-4 pb-lg-5 mt-4 mt-lg-5 bcl-footer__bordered-row row',
+    'cols': [
+      {
+        'classes': 'col-12 col-lg-4 pb-4',
+        'items': [
+        {
+          'type': 'content',
+          'content': drupal_block('whitelabel_eu_logo_block'),
```

Comment:
you are deleting the logos in the header refactor so you need to add the logo directly